packages/components/src/components/themed-scrollbars/themed-scrollbars.scss

```diff
@@ -1,7 +1,6 @@
 /** @define dc-themed-scrollbars */
 .dc-themed-scrollbars {
     overflow-y: auto;
-    overflow-x: hidden;
```

Comment:
Why is this being removed? if `&--has-horizontal` `className` is applied by setting either `horizontal` prop (has or only), it should apply the  `overflow-x: auto` rules.

Reply:
Ok, I added this line back and moved the scrollbars from the tables to the entire reports page for better UX